FoodBookApp/Handlers/NotificationHandler.swift

```diff
@@ -15,6 +15,7 @@ class NotificationHandler {
         { success, error in
             if success {
                 print("Access granted!")
+                self.sendLastReviewNotification(date: Date())
```

Comment:
I know this is not that big of a deal but since you wanted to change this, I am marking it for change